packages/http-framework-i18n/README.md

```diff
@@ -13,10 +13,7 @@ import { addFormatters, init, load } from '@skyra/http-framework-i18n';
 await load(new URL('/locales', import.meta.url));
 
 // Add formatters, those will be added in `i18next.services.formatter`:
-addFormatters(
-	{ name: 'uppercase', format: (value) => value.toUpperCase() },
-	{ name: 'lowercase', format: (value) => value.toLowerCase() }
-);
+addFormatters({ name: 'uppercase', format: (value) => value.toUpperCase() }, { name: 'lowercase', format: (value) => value.toLowerCase() });
```

Comment:
Can you add a `//` after the first object, so they're split in two lines for readability?